spec/factories/entry.rb

```diff
@@ -0,0 +1,26 @@
+FactoryBot.define do
+  factory :exempt, class: 'Entry' do
+    name { 'chocolate' }
+    quantity { '1' }
+    price { '10.00' }
+    basic_tax { '0' }
+    import_tax { '0' }
+    final_price { '0' }
+  end
+  factory :imported, class: 'Entry' do
+    name { 'imported bottle of perfume' }
+    quantity { '1' }
+    price { '47.50' }
+    basic_tax { '0' }
+    import_tax { '0' }
+    final_price { '0' }
+  end
+  factory :importedExempt, class: 'Entry' do
+    name { 'imported box of chocolates' }
+    quantity { '1' }
```

Comment:
Tip: You can put the repetitive attribute in the main factory, and then use `traits` https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#traits